src/transaction_builder.js

```diff
@@ -408,7 +408,7 @@ TransactionBuilder.prototype.sign = function (index, keyPair, redeemScript, hash
     input.signatures[i] = signature
 
     return true
-  }, this), 'key pair cannot sign for this input')
```

Comment:
`this` context here was superfluous, probably shouldn't be in this commit, sorry.  Will remove if necessary.